pages/accounts.js

```diff
@@ -30,6 +30,29 @@ export default function Accounts() {
       return false;
     }
   });
+  const savingsAccount = user?.accounts?.find(account => {
+    if (account.name === "Savings account") {
```

Comment:
This will crash if `user` or `user.accounts` is undefined.
You are using conditional chaining so you expect that this case can occur and you should handle it appropriately, e.g.:

```js
const savingsAccount = user?.accounts ? user.accounts.find(...) : null;
```